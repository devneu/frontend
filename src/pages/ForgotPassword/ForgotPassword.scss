$border-size: 5px 0px 0px 5px;
$max: 100%;

.forgot-password {
  padding: 50px;
  position: fixed;
  height: 100%;
  width: 100%;
  overflow-y: auto;
  background-color: #0050b3;

  .img-block {
    border-radius: $border-size;
    img {
      width: $max;
      height: $max;
      object-fit: cover;
      object-position: 10% 30% 50% 50%;
      border-radius: $border-size;
    }
  }

  .form-block {
    background-color: white;
    border-radius: 0px 5px 5px 0px;
    display: flex;
    justify-content: center;
    text-align: center;

    .title {
      font-weight: normal;
      margin-top: 40px;
    }

    .secondary-text {
      font-size: 16px;
    }

    .email {
      border-radius: 25px;
      height: 50px;
      max-width: 345px;
      margin-top: 20px;
    }

    button {
      &.reset-password {
        height: 45px;
        max-width: 345px;
      }
    }

    .line {
      height: 0.1px;
      background-color: lightgray;
      border: none;
    }

    .ant-btn-dangerous {
      background-color: #f5222d;
      color: white;

      &:hover {
        background-color: #cf1322;
      }
    }

    .footer {
      text-align: center;
      display: flex;
      flex-direction: column;
      text-align: center;
      a {
        margin: 5px;
      }
    }
  }
}
