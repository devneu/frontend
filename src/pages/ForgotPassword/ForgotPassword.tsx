import React from 'react';

const ForgotPassword = () => <div>Forgot password page</div>;

export default ForgotPassword;
