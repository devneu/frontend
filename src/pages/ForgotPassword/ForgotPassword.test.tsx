import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ForgotPassword from './ForgotPassword';

describe('ForgotPassword', () => {
  it('Should successfully render', () => {
    const wrapper = renderer
      .create(
        <BrowserRouter>
          <ForgotPassword />
        </BrowserRouter>,
      )
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
