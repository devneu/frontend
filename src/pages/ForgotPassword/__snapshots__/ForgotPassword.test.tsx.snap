// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPassword Should successfully render 1`] = `
<div
  className="forgot-password"
>
  <div
    className="ant-row ant-row-center"
    style={Object {}}
  >
    <div
      className="ant-col img-block ant-col-xs-0 ant-col-sm-0 ant-col-md-10 ant-col-lg-8 ant-col-xl-8 ant-col-xxl-6"
      style={Object {}}
    >
      <img
        alt="Welcome on portal"
        src="https://i.imgflip.com/5jwx15.jpg"
      />
    </div>
    <div
      className="ant-col form-block ant-col-xs-24 ant-col-sm-24 ant-col-md-10 ant-col-lg-8 ant-col-xl-8 ant-col-xxl-6"
      style={Object {}}
    >
      <div
        className="ant-col ant-col-18"
        style={Object {}}
      >
        <h3
          className="ant-typography title"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Forgot Your Password?
        </h3>
        <span
          className="ant-typography ant-typography-secondary secondary-text"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          We get it, stuff hapens. Just enter your email address below and we\`ll send you a link to reset your password!
        </span>
        <form
          className="ant-form ant-form-horizontal"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <input
                    className="ant-input email"
                    name="email"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    placeholder="Enter Email Address..."
                    type="text"
                    value=""
                  />
                  
                </div>
              </div>
            </div>
          </div>
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <button
                    className="ant-btn ant-btn-primary ant-btn-round ant-btn-block reset-password"
                    onClick={[Function]}
                    type="submit"
                  >
                    <span>
                      Reset Password
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
          <hr
            className="line"
          />
          <div
            className="footer"
          >
            <a
              aria-current={null}
              href="/sign-out"
              onClick={[Function]}
            >
              Create an Account
            </a>
            <a
              aria-current={null}
              href="/sign-in"
              onClick={[Function]}
            >
              Already have an account? Login!
            </a>
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;
